app.js

```diff
@@ -166,8 +170,7 @@ function LeftButton(route, navigator, index, navState) {
         <TouchableOpacity
           style={[styles.backButton, {marginLeft: 10, marginVertical: Platform.OS === 'android' ? 21 : 16}]}
           onPress={() => {
-            navigator.pop()
-            settingsButtonActive = false
+            closeSettings()
           }}
```

Comment:
Almost

We need to give `navigator` to the function

`onPress={() => closeSettings(navigator)}`